allennlp/models/vision_text_model.py

```diff
@@ -106,42 +106,33 @@ def from_huggingface_model_name(
     ):
         transformer = AutoModel.from_pretrained(model_name)
 
-        text_embeddings = deepcopy(transformer.embeddings)
-
         # Albert (and maybe others?) has this "embedding_size", that's different from "hidden_size".
         # To get them to the same dimensionality, it uses a linear transform after the embedding
         # layer, which we need to pull out and copy here.
         if hasattr(transformer.config, "embedding_size"):
             config = transformer.config
 
+            text_embeddings = TransformerEmbeddings.from_pretrained_module(
+                transformer.embeddings, output_size=config.hidden_dim
+            )
+
             from transformers.models.albert.modeling_albert import AlbertModel
 
             if isinstance(transformer, AlbertModel):
-                linear_transform = deepcopy(transformer.encoder.embedding_hidden_mapping_in)
+                text_embeddings.linear_transform = deepcopy(
```

Comment:
Usually I don't pay a lot of attention to this, but Codecov tells me that this line is not covered by tests. It seems somewhat important that the Albert case be covered in tests, since it is so different from the other transformers.